### ISSUE TITLE
Video gets randomly sped up until I change (any) setting

### ISSUE THREAD

**Issue by dimii27** [WofWca/jumpcutter#168]
Every minute or so the playback goes into silence speed but without making the visualised audio red. To fix this I just press any of the keyboard shortcuts so the extension wakes up from drunkness. If I do nothing it might fix by itself after 10-20 seconds, but then I have to seek backwards into the video as I missed some information. It is possible that this happens when my RAM gets near full as I have many tabs open, but I have 16gb RAM on my laptop. Also CPU performance should't be an issue because I have an AMD R7 5000 series on my laptop

**Comment by WofWca** [comment]
Haha, thanks for reporting.

Could you tell me what website you're using, and which value of this setting you have selected?

![image](https://github.com/WofWca/jumpcutter/assets/39462442/7723a37b-cebc-4ea8-a2de-89a42a1afe23)

**Comment by dimii27** [comment]
Yes, chrome on youtube. The issue occurs when using 4x speed for silence, but when I set it to 3x it still occurs but it fixes itself after a second or two

**Comment by WofWca** [comment]
Hmmm this is weird...
Also which "algorithm" are you using, please, and can you also check the version of the extension on chrome://extensions/ ? Is it `1.26.9`?

Do you think this can be related to ads?

Also, just to double check, you're not using any other playback speed-controlling extensions, right?

**Comment by dimii27** [comment]
Extension version - `1.26.9`

Conflicting extensions that I use together:

- ['Improve YouTube!' 🎧 (for YouTube & Videos) 4.826](https://chromewebstore.google.com/detail/improve-youtube-%F0%9F%8E%A7-for-yo/bnomihfieiccainjcjblhegjgglakjdd)
![image](https://github.com/WofWca/jumpcutter/assets/50188164/cd5de33e-e93c-45bf-a753-a4eebb025bd6) 

- [Enhancer for YouTube™ 2.0.123](https://chromewebstore.google.com/detail/enhancer-for-youtube/ponfpcnoihfmfllpaingbgckeeldkhle)
![image](https://github.com/WofWca/jumpcutter/assets/50188164/a9c32cc7-6fbf-495f-830b-a15be429c02e)

- [Tweaks for YouTube 3.64.1](https://chromewebstore.google.com/detail/tweaks-for-youtube/ogkoifddpkoabehfemkolflcjhklmkge)
![image](https://github.com/WofWca/jumpcutter/assets/50188164/a2a48744-5cad-4060-8701-f8c77f82f684)
  - `Jump Cutter` prevents this extension from working propperly unless I click on both extensions and reload the page
  - This extension prevents `Jump Cutte`r from detecting media if I use any of its audio features
  - I would like to use this extension together with `Jump Cutter`  because it can be used to customise time display: [setting](https://github.com/WofWca/jumpcutter/assets/50188164/94de7e59-0610-4435-862d-5310b748a7b9), [explaination](https://github.com/WofWca/jumpcutter/assets/50188164/5c74cec7-7502-4655-ad82-cd78fa2acafd) - I can see remaining the remaining time of the video adjusted to current playback speed (I dream about understanding the code of both extensions so I can add a time display that is calculated according to `Jump Cutter`'s Time saved info. But I am no programmer)

- **[Video Speed Controller 0.7.3](https://chromewebstore.google.com/detail/video-speed-controller/nffaoalbilbmmfgbnbgppjihopabppdk)** (!!!)
![image](https://github.com/WofWca/jumpcutter/assets/50188164/6251a591-a51a-4424-8ca4-df944e1d9abf)

Adblock:
- [uBlock Origin 1.57.0](https://chromewebstore.google.com/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm) - with all the bells and whistles
![image](https://github.com/WofWca/jumpcutter/assets/50188164/b6c14ef7-54ae-4838-9f89-d156d152be71)


I will try to troubleshoot this further to find if it gets fixed when I disable these extensions [will update]

**Comment by WofWca** [comment]
Oh my. That's a lot of extensions.
I do strive to make mine compatible with others, so yeah, please do let me know if you manage to narrow down the issue. Thanks!

**Comment by dimii27** [comment]
Long story short: I couldnțt find a specific extension that causes my problems. Instead I came to the conclusion that it's either using too many extensions in general, too many tabs open or both. I'm afraid to close my tabs, but incognito mode does show a signifiant increase in browser performance using [this benchmark](https://browserbench.org/Speedometer3.0/). Might update when I close all my tabs, but that seems to be it. Even at less than 90% RAM usage I believe the performance has to suffer.

**Comment by dimii27** [comment]
![image](https://github.com/WofWca/jumpcutter/assets/50188164/b9b918ea-8b40-4e74-abfc-8fc7d1b33ca6)

 I seem to have mostly fixed the problems by using these settings and disabling everything but this extension:
 [Tweaks for YouTube 3.64.1](https://chromewebstore.google.com/detail/tweaks-for-youtube/ogkoifddpkoabehfemkolflcjhklmkge)
![image](https://github.com/WofWca/jumpcutter/assets/50188164/4c82b32f-ad0d-4b05-832c-dc46e0a73ed9)

but any setting within `tweaks for youtube` that affects sound volume should be turned off. I'm too distracted to find another combination that works, but this is good enough for me. Also, I think that video speed controller, while it mostly works well, might get the video stuck on the sped up version of speed. 

P.S. in the mean time I have matured more and I have been focusing more on actually studying and not tweaking my browser settings. it helps if I know for sure what is essential and what's not for my studying so when I notice I have like 20 tabs open, I select the two of them that actually matter and `Right_Click` > `Close other tabs`. 

P.P.S. I'm looking forward to find a way to adapt the `remaining time` displayed with `Tweaks for YouTube` (using this secret spell: `%current_time / %duration (%pr_remaining_time) <?pr_not_1 %playback_rate ?>`) by the average speed calculated by `JumpCutter` 
![image](https://github.com/WofWca/jumpcutter/assets/50188164/c657f95c-3cf4-4c2f-98e6-d74870161f3b) - average speed by jumpcutter
![image](https://github.com/WofWca/jumpcutter/assets/50188164/525e1508-a0ab-40fb-b66f-3ccf07741d0d) - remaining time in the video by `Tweaks for YouTube`

to go further down the rabbit hole, I could then find a way to substract SponsorBlock segments from this, but that means constantly calculating the total time of the segments on the right hand. this could be an unrealistic and unnecessary dream, too much for an average poor computer to handle, especially if it's on Chrome and along with 20+ other tabs. when I was writing my previous comments I had laying around around 60 tabs open (I am not exaggerating) that I was afraid to close for several weeks. The hoarding is real
